source/index.html

```diff
@@ -1,39 +1,282 @@
-<!doctype html>
+<!DOCTYPE html>
 <html>
```

Comment:
The `html` tag should have a `lang` attribute, like: `<html lang="en-US">` … the `en` means "English", the `US` means "United States (Variant)".